docs/source/rbac/use-cases.md

```diff
@@ -9,12 +9,12 @@ To determine which scopes a role should have, one can follow these steps:
 5. Customize the scopes with filters if needed
 6. Define the role with required scopes and assign to users/services/groups/tokens
 
-Below, different use cases are presented on how to use the RBAC framework.
+Below, different use cases are presented on how to use the [RBAC framework](https://jupyterhub.readthedocs.io/en/stable/rbac/index.html).
```

Comment:
Sorry, it should be an _internal_ link. I believe you can use `[RBAC framework](./index.md)`

Reply:
Thank you for the correction, I will make the necessary changes